src/atlases.jl

```diff
@@ -216,11 +240,13 @@ function induced_basis(
     return dual_basis(M, p, A.basis)
 end
 
-"""
+@doc raw"""
     InducedBasis(vs::VectorSpaceType, A::AbstractAtlas, i)
 
 The basis induced by chart with index `i` from an [`AbstractAtlas`](@ref) `A` of vector
-space of type `vs`.
+space of type `vs`, i.e. if you write the chart ``\varphi_i = (\varphi_i^1,...,\varphi_n^1)^{\mathrm{T}}``.
+as its coordinate functions, then the basis consists of the partial derivatives ``\Bigl(\frac{\partial}{\partial\varphi_i^j}\Bigr)_p``.
```

Comment:
I think this might be a bit too informal (see also my comment to the `.md` file).

Reply:
I tried for a while to come up with an idea – I could write a few pages, but that would be too long for a doctoring. Can you provide an idea what you would write?

Reply:
I'll try to think of something. Essentially, ``\Bigl(\frac{\partial}{\partial\varphi_i^j}\Bigr)_p`` IMO doesn't explain too much without  some additional definitions and an explanation why we can define a basis this way -- I'm not even totally sure myself about that notation.

Reply:
I wrote a more extended explanation.

Reply:
Thanks, this looks very nice now :+1: